sorting/Topological_sort.cpp

```diff
@@ -0,0 +1,91 @@
+#include <algorithm>
+#include <iostream>
+using namespace std;
+
+#define MAX 10
+int n, adj[MAX][MAX];
+int front = -1, rear = -1, Queue[MAX];
+
+void CreateGraph() {

```

Comment:
Undocumented function. Please document it.
Also, I think the function name could look better like:

      create_graph()